docs/Chromaticqa-Guide.md

```diff
@@ -0,0 +1,22 @@
+# ChromaticQA
+
+## Introduction to ChromaticQA
+[Chromatic Docs](https://docs.chromaticqa.com/)
+
+ChromaticQA is a visual regression tool for storybook maintained by Storybook.The purpose of having chromaticQA run on all stories is to capture
+unwanted visual changes introduced by every change done to the code base.This reduces the overhead on manual regression as it supports screenshot
```

Comment:
there shouldn't be a line break here, all paragraph should be on line 6

Reply:
same throughout the doc

Reply:
yup will make the changes